### PR TITLE
Null flavor mongoid5

### PR DESCRIPTION
**Pull request by dczulada**

### CODE REVIEW

**Comment by dczulada** [comment]
This still needs tests

File: lib/health-data-standards/import/cda/medication_importer.rb [hunk 1]
```diff
@@ -54,7 +54,8 @@ def create_entry(entry_element, nrh = NarrativeReferenceHandler.new)
         def extract_reason_or_negation(parent_element, medication)
           negation_indicator = parent_element['negationInd']
           if negation_indicator.nil? && parent_element.parent.name == "entryRelationship"
-            super(parent_element.parent.parent, medication)
+            super(parent_element.parent.parent, medication, parent_element)
+            #extract_negated_code(parent_element, medication) if medication.codes.empty?
```
Comment (okeefm) [inline lib/health-data-standards/import/cda/medication_importer.rb]: Is there a reason this commented out code is here?

File: lib/health-data-standards/import/cda/section_importer.rb [hunk 2]
```diff
@@ -173,7 +173,10 @@ def import_person(person_element)
           return person
         end
 
-        def extract_reason_or_negation(parent_element, entry)
+        # extracts the reason or negation data. if an element is negated and the code has a null flavor, a random code is assigned for calculation
+        # coded_parent_element is the 'parent' element when the coded is nested (e.g., medication order)
+        def extract_reason_or_negation(parent_element, entry, coded_parent_element = nil)
+          coded_parent_element = parent_element unless coded_parent_element
```
Comment (okeefm) [inline lib/health-data-standards/import/cda/section_importer.rb]: can replace this with `coded_parent_element ||= parent_element`

**Review by okeefm — CHANGES_REQUESTED** [review]

**Comment by dczulada** [comment]
@okeefm updated

**Review by okeefm — APPROVED** [review]

**Pull request merged**